tests/parser/ttupleunpack.nim

```diff
@@ -0,0 +1,18 @@
+discard """
+  file: "ttupleunpack.nim"
+  output: ""
+  exitcode: 0
+"""
+proc foo(): tuple[x, y, z: int] =
+  return (4, 2, 3)
+
+var (x, _, y) = foo()
```

Comment:
You need to test var (x, _, _) = foo()  instead. My bet: it fails because the codegen generates the same names for the 2 _ identifiers. You need to set sfShadowed for _ or perhaps sfGenSym or sfFromGeneric.